backend/src/main/java/com/github/johannesthorbergsson/backend/bikes/BikeController.java

```diff
@@ -0,0 +1,21 @@
+package com.github.johannesthorbergsson.backend.bikes;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.security.Principal;
+import java.util.List;
+
+@RestController
+@RequestMapping("api/bikes/")
+@RequiredArgsConstructor
+public class BikeController {
+    private final BikeService bikeService;
```

Comment:
Are there plans to implement dependency injection via the constructor? Does accessing the object work if it's not instantiated? For example: 
`public BikeController (BikeService bikeService){this.bikeService = bikeService}`